src/services/services.ts

```diff
@@ -376,7 +376,7 @@ namespace ts {
             super(pos, end);
         }
 
-        get unescapedText(): string {
+        get text(): string {
             return unescapeLeadingUnderscores(this.escapedText);

```

Comment:
Can we cache this since the node is supposed to be immutable? Potentially `return (delete this.text, (this as any).text = unescapeLeadingUnderscores(this.escapedText))` or using an internal field. Not sure which is better.